tests/test_device_types.py

```diff
@@ -101,5 +101,5 @@ def test_equality(self):
     def test_string_representations(self):
         """Test that the string representations are unique."""
         all_device_types = xcrun.simctl.listall.device_types()
-        strings = set([str(device_type) for device_type in all_device_types])
+        strings = set([device_type.__str__() for device_type in all_device_types])
```

Comment:
This should stick as `str(x)` rather than `x.__str__()`. The `str` function calls `__str__` if it exists, but falls back to `__repr__` if not. Since this should be unique in either case, we should stick with not calling dunder methods.